ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+///Properties that make up cell content
+@objc(MSFListVnextCell)
+public class MSFListVnextCell: NSObject, ObservableObject, Identifiable {
+    public var id = UUID()
+    @objc @Published public var leadingView: UIImage?
+    @objc @Published public var title: String = ""
+    @objc @Published public var subtitle: String?
+    @objc @Published public var trailingView: TableViewCellAccessoryType = .none
+    @objc @Published public var titleLineLimit: Int = 1
+    @objc @Published public var subtitleLineLimit: Int = 1
+    @objc @Published public var handler: (() -> Void)?
+}
+
+///Properties that make up list content
+@objc(MSFListVnextState)
+public class MSFListVnextState: NSObject, ObservableObject {
+    @objc @Published public var sectionTitle: String?
+    @objc @Published public var hasBorder: Bool = false
+}
+
+@objc(MSFListIconVnextStyle)
+/// Pre-defined styles of icons
+public enum MSFListIconVnextStyle: Int, CaseIterable {
+    case none
+    case iconOnly
+}
+
+@objc(MSFListCellVnextHeight)
+/// Pre-defined layout heights of cells
+public enum MSFListCellVnextLayoutType: Int, CaseIterable {
+    case oneLine
+    case twoLines
+    case threeLines
+
+    var height: CGFloat {
+        switch self {
+        case .oneLine:
+            return 48
+        case .twoLines:
+            return 64
+        case .threeLines:
+            return 84
+        }
+    }
+}
+
+public class MSFListTokens: ObservableObject {
+    @Published public var backgroundColor: UIColor!
+    @Published public var borderColor: UIColor!
+    @Published public var disclosureIconForegroundColor: UIColor!
+    @Published public var iconColor: UIColor!
+    @Published public var leadingTextColor: UIColor!
+    @Published public var subtitleColor: UIColor!
+    @Published public var trailingItemForegroundColor: UIColor!
+
+    @Published public var highlightedBackgroundColor: UIColor!
+
+    @Published public var borderSize: CGFloat!
+    @Published public var disclosureInterspace: CGFloat!
+    @Published public var disclosureSize: CGFloat!
+    @Published public var horizontalCellPadding: CGFloat!
+    @Published public var iconInterspace: CGFloat!
+    @Published public var iconSize: CGFloat!
```

Comment:
If iconSize and largeIconSize are not going to be used at the same time, we can make this only one var (iconSize) and assign the correct token value accordingly.